src/app/public/modules/country-field/country-field.module.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  NgModule
+} from '@angular/core';
+
+import {
+  CommonModule
+} from '@angular/common';
+
+import {
+  FormsModule,
+  ReactiveFormsModule
+} from '@angular/forms';
+
+import {
+  SkyI18nModule
+} from '@skyux/i18n';
+
+import {
+  SkyAutocompleteModule
+} from '../autocomplete';
+
+import {
+  SkyCountryFieldComponent
+} from './country-field.component';
+
+import {
+  SkyLookupResourcesModule
+} from '../shared';
```

Comment:
This should come before `./country-field.component`.

Reply:
Done